crates/sui-storage/src/indexes.rs

```diff
@@ -143,64 +151,140 @@ impl IndexStore {
     fn get_transactions_by_object<KeyT: Clone + Serialize + DeserializeOwned + PartialEq>(
```

Comment:
we recently added some metrics (latency, counter etc) for event queries, maybe worth doing the same for txns.

Reply:
I will do this in a new PR 

Issue for tracking: https://github.com/MystenLabs/sui/issues/5148

cc/fyi @666lcz @srinath-mysten